docs/benefits.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: WAVS benefits
+---
+<!--import { Callout } from 'fumadocs-ui/components/callout';-->
+<!--import { Tab, Tabs } from "fumadocs-ui/components/tabs";-->
+
+<img alt="WAVS" src="/banners/benefits.png" />
```

Comment:
ideally we use a raw of main over there so these actually load, like `https://raw.githubusercontent.com/Lay3rLabs/WAVS-docs/refs/heads/main/public/start-building.png`